cli.js

```diff
@@ -84,7 +88,8 @@ getKeys((err, keys) => {
     console.error(err)
     process.exit(1)
   }
-  httpProxy
+  const port = argv.port || defaultPort
```

Comment:
(nitpick) I don't think this is necessary because default is set in configuration of the cli. But it's ok to keep it.